calcom/cal.com
# feat: slack app

## What does this PR do?

Create slack app for notifications


Fixes #10683 

 Loom Video: https://www.loom.com/share/c7761e726ecd4db69116c6fa0f65a128

## Type of change




## How should this be tested?



- Create a bot app in slack for messaging
- Install slack in Cal.com by providing valid credentials
- Book a meeting, message should appear in the channel

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes


/claim #10683 

edit - updated the video link
